docs/sources/alerting/unified-alerting/message-templating.md

```diff
@@ -0,0 +1,168 @@
+Status       | string    | `firing` or `resolved`
+Labels       | KeyValue  | A set of labels attached to the alert.
+Annotations  | KeyValue  | A set of annotations attached to the alert.
+StartsAt     | time.Time | Time the alert started firing.
+EndsAt       | time.Time | Only set if the end time of an alert is known. Otherwise set to a configurable timeout period from the time since the last alert was received.
+GeneratorURL | string    | A back link to Grafana or external Alertmanager.
+SilenceURL   | string    | Link to grafana silence for with labels for this alert pre-filled. Only for Grafana managed alerts.
+DashboardURL | string    | Link to grafana dashboard, if alert rule belongs to one. Only for Grafana managed alerts.
+PanelURL     | string    | Link to grafana dashboard panel, if alert rule belongs to one. Only for Grafana managed alerts.
+Fingerprint  | string    | Fingerprint that can be used to identify the alert.
+
+### KeyValue
+
+`KeyValue` is a set of key/value string pairs used to represent labels and annotations.
+
+Annotation example containing two annotations:
+
+```json
+{
+  "summary": "alert summary",
+  "description": "alert description"
+}
+```
+
+In addition to direct access of data (labels and annotations) stored as KeyValue, there are also methods for sorting, removing and transforming.
+
+Name        | Arguments | Returns                                 | Notes
+------------|-----------|-----------------------------------------|----------------
+SortedPairs |           | Sorted list of key & value string pairs |
+Remove      | []string  | KeyValue                                | Returns a copy of the Key/Value map without the given keys.
+Names       |           | []string                                | List of label names
+Values      |           | []string                                | List of label values
+
+### Functions
+
+Some functions to transform values are also available, along with [default functions provided by Go templating](https://golang.org/pkg/text/template/#hdr-Functions).
+
+Name         | Arguments                    | Returns
+-------------|------------------------------|----------------------------------------------
+title        | string                       | Capitalizes first character of each word.
+toUpper      | string                       | Converts all characters to upper case.
+match        | pattern, string              | Match a string using RegExp.
+reReplaceAll | pattern, replacement, string | RegExp substitution, unanchored.
+join         | string, []string             | Concatenates the elements of the second argument to create a single string. First argument is the separator. 
+safeHtml     | string                       | Marks string as HTML, not requiring auto-escaping.
+stringSlice  | ...string                    | Returns passed strings as slice of strings.
+
+## Using templating in contact point fields
```

Comment:
This would be a good topic by itself.

Reply:
My experience with using prometheus/alertmanager docs was that I wanted to see all this info in a single page :thinking: as normally you want to have data reference handy while working on the notification fields. But don't feel strongly, should I split this into multiple topics?

Reply:
Yes, "Message Templating" is reference material indeed. However, "Using templating in contact point fields" has a number of procedures. 

Separately, the topic is too long for easy consumption, can be a distraction for the "naive" users who are our target audience.

Reply:
done!